docs/Deploying/Configuration.md

```diff
@@ -2522,3 +2522,27 @@ If not set then the Prometheus `/metrics` endpoint is disabled.
     basic_auth:
       password: "bar"
   ```
+
+#### Packaging
```

Comment:
The packaging feature is using this flag/feature to force an enroll but I'm not sure the config belongs to "packaging" per-se.

Reply:
yea, agreed it's a tough one. I couldn't imagine an scenario outside packaging so I included it in here. I know you said it's ok but if we change our mind I'm happy to move this where it makes more sense.